stdlib/REPL/test/replcompletions.jl

```diff
@@ -1980,7 +1980,7 @@ let (c, r, res) = test_complete_context("getkeyelem(mutable_const_prop).value.")
 end
 
 # JuliaLang/julia/#51548
-# don't return wrong result due to mutable inconsistentcy
+# don't return wrong result due to mutable inconsistency
```

Comment:
@simeonschaub @aviatesk 

Does this refer to the `:consistent` effect? If so, should this say "don't return wrong result due to mutable in-`:consistent`-cy", or something like that?

Reply:
It does refer to the effect, but IMHO it's fine to refer to the effect like this. Spelling it `consistent`-cy always reads a bit clunky to me